src/constants/routes.js

```diff
@@ -0,0 +1,5 @@
+export const LANDING = '/'
```

Comment:
Oh hey routes!  I was wondering about those!

Reply:
Meh... for now they are just constants... but better than hand writing routes all over the place